### PR TITLE
3.35.2: Adds new SDK versions and contract files

### PR DESCRIPTION
**Pull request by kundadebdatta**
# Pull Request Template

## Description

This bumps the SDK version from `3.35.1` to `3.35.2`. No changes in the GA public and preview contract/s.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -13,6 +13,23 @@ Preview features are treated as a separate branch and will not be included in th
...
+### <a name="3.36.0"/> [3.36.0](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.36.0) - 2023-07-17
+
+#### Fixed
+- [3973](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3973) AI Integration: Fixes event generation for failed requests
+- [3917](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3917) Query: Fixes malformed continuation token exception type and message
+- [3969](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3969) Diagnostics: Fixes verbose levels for "Operation will NOT be retried"
+
+#### Added
+- [3668](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3668) Query : Adds string comparison alternative when converting LINQ to SQL (Thanks [@ernesto1596](https://github.com/ernesto1596))
+- [3956](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3956) Client Encryption: Adds package reference Microsoft.Azure.Cosmos version 3.35.1-preview
```
Comment (ealsur) [inline changelog.md:30]: This is not related to our SDK

File: changelog.md [hunk 2]
```diff
@@ -13,6 +13,23 @@ Preview features are treated as a separate branch and will not be included in th
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### <a name="3.36.0-preview"/> [3.36.0-preview](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.36.0-preview) - 2023-07-17
```
Comment (ealsur) [inline changelog.md:16]: If there is no public API change, should this be a 3.35.2?

File: changelog.md (hunk 1 above)
Comment (kundadebdatta) [reply changelog.md]: Removed.

File: changelog.md (hunk 2 above)
Comment (kundadebdatta) [reply changelog.md]: Downgraded the version to `3.35.2`

File: changelog.md [hunk 3]
```diff
@@ -13,6 +13,22 @@ Preview features are treated as a separate branch and will not be included in th
...
 
+### <a name="3.35.2-preview"/> [3.35.2-preview](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.35.2-preview) - 2023-07-17
+
+#### Added
+- [3951](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3951) Upgrade Resiliency: Adds Code to Enable Replica Validation Feature By Default for Preview
+
+### <a name="3.35.2"/> [3.35.2](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.35.2) - 2023-07-17
+
+#### Fixed
+- [3973](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3973) AI Integration: Fixes event generation for failed requests
```
Comment (kirankumarkolli) [inline changelog.md:24]: AI -> please expand otherwise might get mis-interpreted.

File: Directory.Build.props [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 	<PropertyGroup>
-		<ClientOfficialVersion>3.35.1</ClientOfficialVersion>
-		<ClientPreviewVersion>3.35.1</ClientPreviewVersion>
+		<ClientOfficialVersion>3.35.2</ClientOfficialVersion>
+		<ClientPreviewVersion>3.35.3</ClientPreviewVersion>
```
Comment (kirankumarkolli) [inline Directory.Build.props:4]:
```suggestion
		<ClientPreviewVersion>3.35.2</ClientPreviewVersion>
```

File: changelog.md (hunk 3 above)
Comment (kundadebdatta) [reply changelog.md]: Updated the verbiage.

File: Directory.Build.props (hunk 4 above)
Comment (kundadebdatta) [reply Directory.Build.props]: Thanks for pointing this! Fixed.

File: changelog.md [hunk 5]
```diff
@@ -13,6 +13,22 @@ Preview features are treated as a separate branch and will not be included in th
...
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### <a name="3.35.2-preview"/> [3.35.2-preview](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.35.2-preview) - 2023-07-17
+
+#### Added
+- [3951](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3951) Upgrade Resiliency: Adds Code to Enable Replica Validation Feature By Default for Preview
+
+### <a name="3.35.2"/> [3.35.2](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.35.2) - 2023-07-17
+
+#### Fixed
```
Comment (sourabh1007) [inline changelog.md:23]: this should go to preview section...it is preview feature

Comment (kundadebdatta) [reply changelog.md:26]: Moved to preview.

**Review by sourabh1007 — APPROVED** [review]
